File: src/main/Java/boj_step3/boj25314.java
import java.io.*;

public class boj25314 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        String longStr = "long ";
        writer.append(longStr.repeat(n / 4) + "int");
        writer.close();
    }
}

File: src/main/kotlin/boj25314.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    for (i in 1 .. (n / 4))
        w.write("long ")
    if(n % 4 != 0) w.write("long ")
    w.write("int")
    w.close()
}